Classes/UT3LeviathanBolt.uc

```diff
@@ -34,7 +35,8 @@ defaultproperties
 	Damage=100
 	DamageRadius=300
 	MomentumTransfer=4000
-	
+        LifeSpan=4.0
```

Comment:
There is one space too much here

Reply:
The extra space isn't there on edit nor my class file and I can't re save it here without making some kind of change :S

Reply:
Try removing one space from the ones that are there, perhaps?..